src/VisualStudio/IntegrationTests/VisualBasic/BasicBuild.cs

```diff
@@ -0,0 +1,39 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using Roslyn.VisualStudio.Test.Utilities;
+using Xunit;
+
+namespace Roslyn.VisualStudio.Integration.UnitTests
+{
+    [Collection(nameof(SharedIntegrationHost))]
+    public class BasicBuild
+    {
+        private IntegrationHost _host;
+
+        public BasicBuild(IntegrationHost host)
+        {
+            _host = host;
+            _host.Initialize();
```

Comment:
Yes, the factory method is a good approach. That way you can't forget to call Initialize. I've done that pattern in non-Roslyn projects and liked it very much.

(use a method, not a property...properties are supposed to be cheap and launching VS most definitely isn't!)